app/grid.scss

```diff
@@ -1,133 +1,154 @@
 @font-face {
-  font-family: Fontin Regular;
-  src: url(fonts/fontin2_pc/Fontin-Regular.ttf);
+    font-family: Open_Sans;
+    src: url(fonts/Open_Sans/OpenSans-Regular.ttf);
 }
 
 @font-face {
-  font-family: Muli;
-  src: url(fonts/Muli/Muli-Regular.ttf);
+    font-family: Merriweather-Regular;
+    src: url(fonts/merriweather/Merriweather-Regular.ttf);
 }
 
 body {
-  margin: 0;
-  height: 100%;
-  width: 100%;
-  font-family: Arial, Roboto, sans-serif;
+    margin: 0;
+    height: 100%;
+    width: 100%;
+    font-family: Roboto;
 }
 
-h2 {
-  text-align: left;
-  color: rgb(45, 45, 45);
-  margin-bottom: 2px;
-  margin-top: 40px;
-  font-family: Roboto;
+#name {
+    font-size: 27px;
+    font-weight: bold;
+    margin-left: 10px;
+    padding-top: 20px;
 }
 
-.grid {
-  display: block;
+#title {
+    margin-left: 10px;
+    margin-top: 20px;
+    font-size: 13px;
+    font-weight: bold;
+    font-family: Roboto;
 }
 
-#main {
-  color: #fff;
-  font-size: 16px;
-  height: 100%;
-  width: 100%;
-  margin: 0;
+h4 {
+    text-align: left;
+    color: rgb(45, 45, 45);
+    margin-bottom: 20px;
+    margin-top: 40px;
+    font-family: Roboto;
+    font-size: 22px;
 }
 
-.menu {
-  background-color: rgb(255, 255, 255);
-  height: 5%;
+.grid {
+    display: block;
 }
 
-#search-bar {
-  text-align: right;
-  font-size: 30px;
-  margin: 0 auto;
-  color: rgb(73, 73, 73);
+#main {
+    display: flex;
+    flex-direction: row;
+    color: #fff;
+    font-size: 16px;
+    height: 100vh;
+    width: 100%;
+    margin: 0;
 }
 
-.profile {
-  background-color: rgb(16, 13, 16);
-  height: 15%;
+#sidebar {
+    background-color: black;
+    width: 400px;
+    height: auto;
 }
 
-.profile .grid {
-  margin-top: 110px;
+#sidebar .menu {
+    display: none;
+    background-color: rgb(255, 255, 255);
+    height: 30px;
 }
 
-#name {
-  font-size: 27px;
-  font-weight: bold;
-  margin-left: 10px;
-  padding-top: 20px;
+#search-bar {
+    text-align: right;
+    font-size: 30px;
+    padding: 15px 20px 0px 0px;
+    color: rgb(73, 73, 73);
+    font-size: 20px;
 }
 
-#title {
-  margin-left: 10px;
-  font-size: 13px;
-  font-weight: bold;
-  font-family: Roboto;
+#profile {
+    margin: 0;
 }
 
 #links {
-  margin-left: 10px;
+    margin-left: 10px;
 }
 
 .link {
-  padding: 3px;
-  font-size: 20px;
+    padding: 3px;
+    font-size: 20px;
 }
 
 #picture {
-  margin: 15px;
-  border: 3px solid #ffffff;
+    margin: 15px;
+    border: 3px solid #ffffff;
 }
 
 #picture {
-  height: 100px;
-  width: 100px;
-  background-color: rgb(216, 21, 21);
-  border-radius: 100%;
+    height: 100px;
+    width: 100px;
+    background-color: rgb(216, 21, 21);
+    border-radius: 100%;
```

Comment:
I'd suggest having a class `circle` that just has `border-radius: 100%` since it might be useful for other things later on and that once we have more than 1 picture, this `#picture` rule might disappear